pyquil/tests/test_job_results.py

```diff
@@ -17,3 +21,16 @@ def test_octet_bits():
     assert [0, 1, 0, 0, 0, 0, 0, 0] == _octet_bits(0b10)
     assert [1, 0, 1, 0, 0, 0, 0, 0] == _octet_bits(0b101)
     assert [1, 1, 1, 1, 1, 1, 1, 1] == _octet_bits(0b11111111)
+
+
+def test_get_is_called():
+    # check that JobResult.is_done() calls .get() on QPU prior to
+    # returning result
+    qpu_connect = QPUConnection('test')
+    job_result = JobResult(qpu_connect, False)
+    try:
```

Comment:
ah I fixed it by setting self.result in the JobResult instantiation, commiting now